src/main/resources/application.yml

```diff
@@ -156,8 +156,8 @@ application:
 
 ### proxy ###
 proxy:
-  host: ${SURICATE_PROXY_URL:}
```

Comment:
Why ?

Reply:
This configuration shouldn't be in application.yml but in application-generic.yml.

I'll push the change.
I will also delete this line in application-generic.yml 
    'driverClassName: com.mysql.jdbc.Driver'

Because it's inserted by the specific database properties.

Reply:
Ok, can u push it on this PR

Reply:
I have also deleted the line active.profiles in application-generic.yml, this line is used to activate others properties.

Like so our properties will be properly set :)

Reply:
The changes has been pushed